core/cautils/fileutils.go

```diff
@@ -325,15 +326,14 @@ func GetFileFormat(filePath string) FileFormat {
 }
 
 // handleListObject handle a List manifest
-func handleListObject(obj workloadinterface.IMetadata) []workloadinterface.IMetadata {
+func handleListObject(list *listworkloads.ListWorkloads) []workloadinterface.IMetadata {
```

Comment:
suggestion: let’s rename this function to something more descriptive

The current name poorly describes what the function does, so it’s hard to understand how exactly it handles the List object at a glance. So let’s leave the function better than we’ve found it and give it a more descriptive name and comment.

How about `listItemsToMetadatas`, since it transforms a List object’s items to a list of `IMetadata` objects? Then the comment would read:
> listItemsToMetadataList transforms items of a given K8s List object to a slice of IMetadata objects

What do you think?